app/src/main/java/org/schabi/newpipe/player/helper/PlayerDataSource.java

```diff
@@ -80,17 +83,6 @@ public PlayerDataSource(@NonNull final Context context, @NonNull final String us
         return new DefaultDashChunkSource.Factory(dataSourceFactory);
     }
 
-    public HlsMediaSource.Factory getHlsMediaSourceFactory() {
```

Comment:
Removed because I don't think there is caching for livestreams. Let me know if otherwise to revert.